### PR TITLE
issues - 2028 - top parameter values added

### PR DESCRIPTION
**Pull request by lovethakker02**
top parameter min and max values were missing. Added details.

### CODE REVIEW

**Comment by Lauragra** [comment]
@piotrci , please review and approve.

**Comment by baywet** [comment]
fixes #2028

File: concepts/query-parameters.md [hunk 1]
```diff
@@ -369,6 +369,8 @@ Use the `$top` query parameter to specify the page size of the result set.
 
 If more items remain in the result set, the response body will contain an `@odata.nextLink` parameter. This parameter contains a URL that you can use to get the next page of results. To learn more, see [Paging](./paging.md). 
 
+You can provide values from 1 to 999 for top parameter.
```
Comment (baywet) [inline concepts/query-parameters.md]:
```suggestion
$top accepts a minimum value of 1 and a maximum value of 999 (inclusive).  
```

**Comment by baywet** [comment]
@lovethakker02 thank you for the contribution and sorry it took so long to get reviewed!
@Lauragra when you merge this PR can you close #2028 please? the commit message didn't include the issue ID and I doubt it'll be closed automatically. Thanks!

**Comment by lovethakker02** [comment]
> @lovethakker02 thank you for the contribution and sorry it took so long to get reviewed!
> @Lauragra when you merge this PR can you close #2028 please? the commit message didn't include the issue ID and I doubt it'll be closed automatically. Thanks!

@baywet - thanks for review. I am just asking for curiosity - did this take same amount of time every-time or there were any issue in my commit?

**Comment by baywet** [comment]
@lovethakker02 no specific issue on your commit, it kind of feel through the cracks on our end for multiple reasons. We're currently working on improving our processes.

**Pull request merged**